server/com.dexels.navajo.compiler.tsl/src/com/dexels/navajo/compiler/tsl/internal/BundleQueueComponent.java

```diff
@@ -77,6 +78,8 @@ public void enqueueScript(final String script, final String path) {
 		        if (!failures.isEmpty()) {
 		            logger.info("Script compilation failed: {}", script);
 		        }
+		        checkOnDemandScriptDependencies(script);
+		        enqueueDependentScripts(script);
```

Comment:
Do we need to do something with dependent scripts on a deletion? Maybe when (eg) entity B gets deleted which is a super entity of entity A, A needs to be uninstalled too?